openedx/core/djangoapps/service_status/tasks.py

```diff
@@ -4,12 +4,11 @@
 
 import time
 
+from celery import task
 from dogapi import dog_stats_api
 
-from djcelery import celery
 
-
-@celery.task
+@task
```

Comment:
Should we be using `shared_task` here instead? http://docs.celeryproject.org/en/3.1/django/first-steps-with-django.html#using-the-shared-task-decorator